src/ProjectTempoListener.cpp

```diff
@@ -0,0 +1,42 @@
+#include "ProjectTempoListener.h"
```

Comment:
As I said, this header file isn't needed.  Just put all in the .cpp file.

There are many examples already of .cpp files without companion .h files, when registration at startup time is enough to cause the code to be visited.

For instance in src/export